src/main/resources/portal.properties.EXAMPLE

```diff
@@ -234,6 +234,7 @@ oncoprint.hotspots.default=true
 # Disable cache (for non authorized portals this can sometimes avoid running
 # out of memory with relatively little performance loss)
 # mybatis.cache.enabled=false
+ehcache.disable_all_caching=false
```

Comment:
maybe follow the convention for the mybatis property? something like `ehcache.cache.disabled`

Reply:
I've renamed it to ehcache.cache_enabled and used underscores for other settings like this (trying to standardize) :
ehcache.xml.configuration -> ehcache.xml_configuration
ehcache.enable.statistics -> ehcache.statistics_enabled
cache.statistics.endpoint.enabled -> ehcache.statistics_endpoint_enabled
ehcache.persistence.path -> ehcache.persistence_path

Reply:
after talking with @averyniceday I went back to cache.statistics_endpoint_enabled